lessons/python/scikit-learn/lesson.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: Machine Learning with scikit-learn
+visible: true
+tags:
+	- python
+	- intermediate
+---
+<!-- change visible to true if you want it on the site -->
+
+# Machine learning in Python
+
+ - **Authors**: Nil Sahin
+ - **Research field**: Computational Biology and Molecular Genetics
+ - **Lesson topic**: Machine Learning using Python's Sklearn packages
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/misc/scikit-learn>
```

Comment:
Uff, silly me. Sorry about that, I meant `lessons/python/scikit-learn`. My mistake!